doc/object_properties.adoc

```diff
@@ -14,61 +14,72 @@ WARNING: This is missing the default values for everything
 
 === `hp_max`
 * internal type: `u16`
+* default value: `1`
 
 Sets the max HP of a player.
 Clamped if it goes above the `u16` max value or under the `u16` min value.
 
 === `breath_max`
 * internal type: `u16`
+* default value: `0`
 
 Sets the max breath of a player.
 
 === `eye_height`
 * internal type: `f32`
+* default value: `1.625`
 
 Sets the camera height. Measured in nodes above the player's feet.
 
 === `zoom_fov`
 * internal type: `f32`
 * units: `degrees`
+* default value: `0.0`
 
 Sets the field of view of the player when they zoom, in degrees. Setting to `0` disables zooming.
 
 == Properties Applying to All Objects
 
 === `physical`
 * field type: `bool`
+* default value: `false`
 
 Controls whether the object collides with nodes that are `walkable`.
 
 === `collide_with_objects`
 * field type: `bool`
+* default value: `true`
 
 Controls whether the object collides with other objects.
 
 CAUTION: `physical` must be set to `true` for this to take effect!
 
+NOTE: This works with players as well but you may expericence https://github.com/minetest/minetest/issues/11783[strange bugs].
+
 === `collisionbox`
 //TIP: This could maybe be linked somewhere else.
-
 * field type: `cuboid_def`
+* default value: `{-0.5, -0.5, -0.5, 0.5, 0.5, 0.5}`
```

Comment:
Is the Lua API wrong? https://github.com/minetest/minetest/blob/master/doc/lua_api.txt#L7174-L7175